src/site/content/en/blog/coop-coep/index.md

```diff
@@ -237,16 +239,17 @@ Here is what you need to do depending on the nature of the resource:
     * Use the `crossorigin` attribute in the loading HTML tag if the resource is
       served with CORS. (For example, `<img src="***" crossorigin>`.)
     * Ask the owner of the resource to support either CORS or CORP.
-* For iframes, use CORP and COEP headers as follows:
-  `Cross-Origin-Resource-Policy: same-origin` (or `same-site`, `cross-origin`
-  depending on the context) and `Cross-Origin-Embedder-Policy: require-corp`.
+* For iframes and worker scripts, set the `Cross-Origin-Resource-Policy:
```

Comment:
Setting CORP is needed only for cross-origin iframe. Not really useful for workers. They are always same-origin.
So, I would remove the "and worker scripts".

COEP on the other site is needed or useful for every environments created by the document. You moved the section later. This is discussed separately, which works.

Reply:
You can only spawn a same-origin worker at the first level, but can load a cross-origin script via that worker if you use `importScripts`. In that case, you have to use `CORP: cross-origin` header. You can see it working at https://cross-origin-isolation.glitch.me/?coep=require-corp&

Reply:
Hmm, this contradicts 🤔
https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#spawning_subworkers

Reply:
I understood "worker script" as the "worker's main response".

This is right worker can then use fetch/importScript to load any resources and those resources are subject to CORP checks. Yes, you can nested DedicatedWorker.

Reply:
In that case, how about phrasing the line like this:

```
For iframes and cross-origin worker scripts loaded via `importScripts`, set the `Cross-Origin-Resource-Policy:
```

Reply:
This feels weird, because it would mean this applies exclusively to those two resources, which is wrong; it applies to every resources.
Also, I feel weird putting on the same level the main resources (the iframe's response) and a subresource (an external script loaded from the worker).

Maybe you can discuss about all subresources, indepedently of the context (window or worker), and give some examples of subresources?
---

- From a `COEP:require-corp` document or a `COEP:require-corp` worker, cross-origin subresources loaded without CORS must set `Cross-Origin-Resource-Policy: cross-origin` header to opt-in being embedded. For instance, this applies to: `<script>`, `importScript`, `<link>`, `<video>`, `<iframe>`, etc...

Reply:
That's a great idea. I'll create a pull request so we can continue the discussion there.

Reply:
https://github.com/GoogleChrome/web.dev/pull/8239